### PR TITLE
github: add bug report template

### PR DESCRIPTION
**Pull request by Rustin-Liu**
I want to report a bug to bot, but it seems we don't have a bug report template.
So I add a template to report bug.

### CODE REVIEW

**Comment by Rustin-Liu** [comment]
@you06 @AndreMouche  PTAL~ Thanks!

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+- Which project: <!-- TiKV/TiDB/... -->
+
+- PR or Issue Link: <!-- pingcap/tidb#18217 -->
+
+### Steps to reproduce(Optional)
+
+### What is expected?
+
+### What is actually happening?
```
Comment (you06) [inline .github/ISSUE_TEMPLATE/bug-report.md:13]: Please add a blank line.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+- [ ] I have searched the [issues](https://github.com/pingcap-incubator/cherry-bot/issues) of this repository and believe that this is not a duplicate.
+
+### Related link
+
+- Which project: <!-- TiKV/TiDB/... -->
+
+- PR or Issue Link: <!-- pingcap/tidb#18217 -->
```
Comment (you06) [inline .github/ISSUE_TEMPLATE/bug-report.md:7]: The example comment may be pingcap-incubator/cherry-bot

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (Rustin-Liu) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Added.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (Rustin-Liu) [reply .github/ISSUE_TEMPLATE/bug-report.md:7]: I think we can keep it. Because the bug reporter always comes from other project.

**Comment by you06** [comment]
Thanks!

**Pull request merged**